src/app/components/cds/inputs/TextField.js

```diff
@@ -81,6 +82,7 @@ const TextField = React.forwardRef(({
                 [styles.outlined]: variant === 'outlined',
                 [styles['input-icon-left']]: iconLeft,
                 [styles['input-icon-right']]: iconRight,
+                [styles['textarea-autoGrow']]: autoGrow,
```

Comment:
@danielevalverde since you are adding this class to the `<textarea` part of this component, the css definition below, needs to target that as well.